sncosmo/salt2utils.pyx

```diff
@@ -13,6 +13,16 @@ from cpython.mem cimport PyMem_Malloc, PyMem_Free
 from libc.math cimport fabs
 from libc.string cimport memcpy
 
+cimport
+numpy as np
```

Comment:
Looks like this broke all of the cimport statements

Reply:
oops - Forgot to test against against a non-built install before pushing. I'll fix that quick.

Reply:
Fixed it.